test/e2e/util.go

```diff
@@ -2128,6 +2128,7 @@ func waitForDeploymentStatus(c clientset.Interface, ns, deploymentName string, d
 		}
 		if totalAvailable < minAvailable {
 			logReplicaSetsOfDeployment(deploymentName, oldRSs, newRS)
+			logPodsOfReplicaSets(c, allRSs, minReadySeconds)
```

Comment:
It's easy to add that test. Just didn't want to spread too many logs everywhere. Will log pods with RSs then.